### PR TITLE
Chaplain Weapon Choice: Hand of God!

### PR DESCRIPTION
**Pull request by ghost**
You can switch out your holy weapon for a scroll that, when used, will teach Hand of God.

Why?
I think a lot of chaplain stuff is feeling cool about it, Hand of God is better at some things but worse at others. 

Pros:
Can't be removed
Looks badass
Has a knockdown
Cons:
Less damage than a weapon
No dismemberment
No holy weapon protection

### CODE REVIEW

**Comment by Armhulen** [comment]
I want to get the arguing out of the way before i put in sleepless nights into this

**Comment by ExcessiveUseOfCobblestone** [comment]
why did you define combos if they don't do anything?

**Comment by Armhulen** [comment]
i just noticed that and removed it my bad

**Comment by ExcessiveUseOfCobblestone** [comment]
Also wow this is a knockdown

**Comment by Armhulen** [comment]
i thought it was fair for 10 damage per hit vs 18

File: code/datums/martial/holy_crunch.dm [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	deflection_chance = 5
+	no_guns = TRUE
+	allow_temp_override = FALSE
+
+/datum/martial_art/chaplain_snap/harm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	A.do_attack_animation(D, ATTACK_EFFECT_PUNCH)
+	var/atk_verb = pick("power word: grail", "power word: justice", "power word: purge", "power word: judicator", "power word: banish")
+	D.visible_message("<span class='danger'>[A] uses [atk_verb] on [D]!</span>", \
+					  "<span class='userdanger'>[A] uses [atk_verb] on you!</span>")
+	D.apply_damage(10, BRUTE)
```
Comment (ChangelingRain) [inline code/datums/martial/holy_crunch.dm]: ignores armor??

File: code/datums/martial/holy_crunch.dm [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	playsound(get_turf(D), 'sound/weapons/punch1.ogg', 25, 1, -1)
+	if(prob(D.getBruteLoss()) && !D.lying)
```
Comment (ChangelingRain) [inline code/datums/martial/holy_crunch.dm]: first attack has a 10% chance to knock down, with each attack increasing it by 10%

Comment (Armhulen) [reply code/datums/martial/holy_crunch.dm]: that's a good comment to add

File: code/datums/martial/holy_crunch.dm (hunk 1 above)
Comment (Armhulen) [reply code/datums/martial/holy_crunch.dm]: there are 18 damage chaplain weapons that pierce armor so it doesn't sound that crazy

File: code/datums/martial/holy_crunch.dm (hunk 2 above)
Comment (Armhulen) [reply code/datums/martial/holy_crunch.dm]: uh huh!

File: code/datums/martial/holy_crunch.dm (hunk 1 above)
Comment (Repukan) [reply code/datums/martial/holy_crunch.dm]: Weapons that can be disarmed or have a tell via sprite or examine.

**Comment by Repukan** [comment]
>Deflection

>No way to disarm

>Guaranteed knockdown the longer a fight goes on

>Makes the chaplain nigh useless during cultist rounds since they become vulnerable to cult magic


It'll fare better as a traitor item to replace His Grace rather than something you can get roundstart.

**Comment by Iamgoofball** [comment]
stop bitching about His Grace

**Comment by Repukan** [comment]
Lmao

**Comment by Armhulen** [comment]
>deflection
5%
>no way to disarm
stun?
>guaranteed knockdown the longer a fight goes on
yes and?
>makes the chaplain nigh useless during cultist rounds
useless fight-wise. you're still able to bless holy water and all that

**Comment by Repukan** [comment]
A big deal about runes is that you can use your rod to erase them, thus crippling their teleportation and so on.

Guaranteed knockdown given by a no discernible weapon that can't be seen and you'd only know you're getting attacked with after you die/during a fight seems plain unfun and it looks like it'd add up quickly with the normal knockdown chance unless it doesn't add up like that.

**Comment by Armhulen** [comment]
from full health, it's 10% more chance every time you hit

it's not a huge stun chance unless you get off a ton of hits

**Comment by Armhulen** [comment]
0 guaranteed knockdowns

**Comment by KorPhaeron** [comment]
>You can switch out your holy weapon for a scroll that, when used, will teach Holy Crunch.

Is this part of the PR I don't see it

**Comment by Supermichael777** [comment]
i'm thinking zelda reference candle with realistic fire hazard action

**Comment by Armhulen** [comment]
@KorPhaeron oh i forgot i'll do it in a few

**Comment by Armhulen** [comment]
and by a few i mean tomorrow hahaha

File: code/datums/martial/holy_crunch.dm [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+		D.visible_message("<span class='warning'>[A] sends [D] to the ground with holy energies!</span>", "<span class='userdanger'>An unseen force sends swipes you off your feet!</span>")
+		D.Knockdown(80)
+	add_logs(A, D, "[atk_verb] (Holy Crunch)")
+	return 1
```
Comment (kevinz000) [inline code/datums/martial/holy_crunch.dm]: return TRUE

**Comment by Armhulen** [comment]
i'll make the martial art stun paper immune but nothing else

File: code/datums/martial/holy_crunch.dm (hunk 1 above)
Comment (Xhuis) [reply code/datums/martial/holy_crunch.dm]: As long as hulks and the officer's saber are a thing, job-specific unarmed attacks that do a lot of damage and pierce armor will always not be as good as them

**Comment by Grazyn** [comment]
monks IN SPACE

File: code/datums/martial/holy_crunch.dm [hunk 4]
```diff
@@ -0,0 +1,40 @@
+/datum/martial_art/holy_crunch
```
Comment (MoreRobustThanYou) [inline code/datums/martial/holy_crunch.dm]: ```/datum/martial_art/holy_crunch```
```/datum/martial_art/chaplain_snap```

**Comment by jughu** [comment]
Pictures

File: code/datums/martial/holy_crunch.dm (hunk 4 above)
Comment (Armhulen) [reply code/datums/martial/holy_crunch.dm]: ty that fixes this

**Comment by Armhulen** [comment]
Sherri Patterson uses power word: purge on Isaac Duncan!

Sherri Patterson uses power word: justice on Isaac Duncan!

Sherri Patterson uses power word: grail on Isaac Duncan!

Sherri Patterson uses power word: banish on Isaac Duncan!

Sherri Patterson sends Isaac Duncan to the ground with holy energies!

Sherri Patterson uses power word: grail on Isaac Duncan!

Sherri Patterson uses power word: judicator on Isaac Duncan!

**Comment by Armhulen** [comment]
AND when used on you:
Sherri Patterson uses power word: justice on you!

Sherri Patterson uses power word: grail on you!

Sherri Patterson uses power word: judicator on you!

Sherri Patterson uses power word: judicator on you!

An unseen force sends swipes you off your feet!

**Comment by Armhulen** [comment]
i broke it zzz i'll fix it zzzzzz

**Comment by Doppeltongue** [comment]
Holy Crunch is an awkward name, is it supposed to be a reference to something? How about something alliterative like say Papal Pugalism? Or maybe Crusading Crush?

**Comment by KorPhaeron** [comment]
holy crunch makes me think of cereal

**Comment by Armhulen** [comment]
I think the name is funny so i'm keeping it

**Comment by Xhuis** [comment]
holycrunch

**Comment by Armhulen** [comment]
fuck that picture is awesome

**Comment by Armhulen** [comment]
guys save on merging for a bit i'm gonna rework this to be more interesting so kor can't troll me

File: code/datums/martial/holy_crunch.dm [hunk 5]
```diff
@@ -0,0 +1,47 @@
+/datum/martial_art/hand_of_god
+	name = "Hand of God"
+	deflection_chance = 0
+	no_guns = TRUE
+	allow_temp_override = FALSE
+
+/datum/martial_art/hand_of_god/harm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	A.do_attack_animation(D, ATTACK_EFFECT_PUNCH)
+	var/attack_type = pick("brute", "burn", "toxin", "knockdown")
+	if(attack_type == "brute")
```
Comment (Xhuis) [inline code/datums/martial/holy_crunch.dm]: I would replace this whole thing with:
```
var/attack_type = list(BRUTE, BURN, TOX, STAMINA)
var/atk_verb = "power word: fist"
switch(attack_type)
   if(BRUTE)
      atk_verb = pick(blah, blah, blah)
      blah
   (same for the other types)
D.apply_damage(10, attack_type)
add_logs(A, D, " used [attack_type] attack (Hand of God) on"
```
You'd remove the switch statements and keep the logging outside of that whole shebang. This seems like a more modular way of doing things, with the exception of the slight buff that makes the knockdown do 10 stamina damage.

**Comment by kevinz000** [comment]
this seems op d00d

File: code/datums/martial/holy_crunch.dm (hunk 5 above)
Comment (kevinz000) [reply code/datums/martial/holy_crunch.dm]: switch statement yeah

File: code/datums/martial/holy_crunch.dm [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+		var/atk_verb = pick("power word: fracture", "power word: break", "power word: crack", "power word: dust", "power word: fissure")
+		D.visible_message("<span class='danger'>[A] uses [atk_verb] on [D]!</span>", \
+					  "<span class='userdanger'>[A] uses [atk_verb] on you!</span>")
+		D.apply_damage(10, BRUTE)
+		playsound(get_turf(D), 'sound/magic/repulse.ogg', 25, 1, -1)
```
Comment (kevinz000) [inline code/datums/martial/holy_crunch.dm]: D

File: code/datums/martial/holy_crunch.dm [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+		add_logs(A, D, " used [atk_verb] (Hand of God) on")
+		return 1
```
Comment (kevinz000) [inline code/datums/martial/holy_crunch.dm]: TRUE

File: code/datums/martial/holy_crunch.dm [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+	if(attack_type == "burn")
+		var/atk_verb = pick("power word: scorch", "power word: sear", "power word: burn", "power word: fireball", "power word: combust")
+		D.visible_message("<span class='danger'>[A] uses [atk_verb] on [D]!</span>", \
+					  "<span class='userdanger'>[A] uses [atk_verb] on you!</span>")
+		D.apply_damage(10, BURN)
+		playsound(get_turf(D), 'sound/magic/teleport_app.ogg', 25, 1, -1)
```
Comment (kevinz000) [inline code/datums/martial/holy_crunch.dm]: D

File: code/datums/martial/holy_crunch.dm [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+		add_logs(A, D, " used [atk_verb] (Hand of God) on")
+		return 1
```
Comment (kevinz000) [inline code/datums/martial/holy_crunch.dm]: TRUE

File: code/datums/martial/holy_crunch.dm [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+	if(attack_type == "toxin")
+		var/atk_verb = pick("power word: impair", "power word: afflict", "power word: debilitate", "power word: hurt", "power word: diminish", )
+		D.visible_message("<span class='danger'>[A] uses [atk_verb] on [D]!</span>", \
+					  "<span class='userdanger'>[A] uses [atk_verb] on you!</span>")
+		D.apply_damage(10, TOX)
+		playsound(get_turf(D), 'sound/effects/bamf.ogg', 25, 1, -1)
+		add_logs(A, D, " used [atk_verb] (Hand of God) on")
+		return 1
```
Comment (kevinz000) [inline code/datums/martial/holy_crunch.dm]: TRUE

File: code/datums/martial/holy_crunch.dm [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+		D.apply_damage(10, BURN)
+		playsound(get_turf(D), 'sound/magic/teleport_app.ogg', 25, 1, -1)
+		add_logs(A, D, " used [atk_verb] (Hand of God) on")
+		return 1
+	if(attack_type == "toxin")
+		var/atk_verb = pick("power word: impair", "power word: afflict", "power word: debilitate", "power word: hurt", "power word: diminish", )
+		D.visible_message("<span class='danger'>[A] uses [atk_verb] on [D]!</span>", \
+					  "<span class='userdanger'>[A] uses [atk_verb] on you!</span>")
+		D.apply_damage(10, TOX)
+		playsound(get_turf(D), 'sound/effects/bamf.ogg', 25, 1, -1)
```
Comment (kevinz000) [inline code/datums/martial/holy_crunch.dm]: D

File: code/datums/martial/holy_crunch.dm [hunk 12]
```diff
@@ -0,0 +1,47 @@
...
+	if(attack_type == "knockdown")
+		var/atk_verb = pick("power word: stasis", "power word: daze", "power word: tranquilise", "power word: benumb", "power word: ban", )
+		D.visible_message("<span class='danger'>[A] uses [atk_verb] on [D]!</span>", \
+					  "<span class='userdanger'>[A] uses [atk_verb] on you!</span>")
+		if(prob(D.getBruteLoss()) && !D.lying)
+			D.visible_message("<span class='warning'>[A] sends [D] to the ground with holy energies!</span>", "<span class='userdanger'>An unseen force sends swipes you off your feet!</span>")
+			D.Knockdown(80)
+			playsound(get_turf(D), 'sound/effects/pray.ogg', 25, 1, -1)
+			add_logs(A, D, " used [atk_verb] (Hand of God) on")
+			return 1
```
Comment (kevinz000) [inline code/datums/martial/holy_crunch.dm]: TRUE

File: code/datums/martial/holy_crunch.dm [hunk 13]
```diff
@@ -0,0 +1,47 @@
...
+		add_logs(A, D, " used [atk_verb] (Hand of God) on")
+		return 1
+	if(attack_type == "knockdown")
+		var/atk_verb = pick("power word: stasis", "power word: daze", "power word: tranquilise", "power word: benumb", "power word: ban", )
+		D.visible_message("<span class='danger'>[A] uses [atk_verb] on [D]!</span>", \
+					  "<span class='userdanger'>[A] uses [atk_verb] on you!</span>")
+		if(prob(D.getBruteLoss()) && !D.lying)
+			D.visible_message("<span class='warning'>[A] sends [D] to the ground with holy energies!</span>", "<span class='userdanger'>An unseen force sends swipes you off your feet!</span>")
+			D.Knockdown(80)
+			playsound(get_turf(D), 'sound/effects/pray.ogg', 25, 1, -1)
```
Comment (kevinz000) [inline code/datums/martial/holy_crunch.dm]: D

File: code/datums/martial/holy_crunch.dm [hunk 14]
```diff
@@ -0,0 +1,47 @@
...
+		playsound(get_turf(D), 'sound/effects/bamf.ogg', 25, 1, -1)
+		add_logs(A, D, " used [atk_verb] (Hand of God) on")
+		return 1
+	if(attack_type == "knockdown")
+		var/atk_verb = pick("power word: stasis", "power word: daze", "power word: tranquilise", "power word: benumb", "power word: ban", )
+		D.visible_message("<span class='danger'>[A] uses [atk_verb] on [D]!</span>", \
+					  "<span class='userdanger'>[A] uses [atk_verb] on you!</span>")
+		if(prob(D.getBruteLoss()) && !D.lying)
+			D.visible_message("<span class='warning'>[A] sends [D] to the ground with holy energies!</span>", "<span class='userdanger'>An unseen force sends swipes you off your feet!</span>")
+			D.Knockdown(80)
```
Comment (kevinz000) [inline code/datums/martial/holy_crunch.dm]: This gives stuntime/meleeattackdelay = 80/8 = 10 free attacks assuming the person being attacked has no stunbreaks/stun resistance which is with its full AP a guarunteed instacrit assuming the person being attacked his no regeneration. You should probably make it lower IMO.

File: code/datums/martial/holy_crunch.dm [hunk 15]
```diff
@@ -0,0 +1,47 @@
...
+			playsound(get_turf(D), 'sound/effects/pray.ogg', 25, 1, -1)
+			add_logs(A, D, " used [atk_verb] (Hand of God) on")
+			return 1
+		D.visible_message("<span class='warning'>[A] tries to drop [D] with holy energies!</span>", "<span class='userdanger'>An unseen force whizzes past you!</span>")
+		playsound(get_turf(D), 'sound/weapons/punchmiss.ogg', 25, 1, -1)
```
Comment (kevinz000) [inline code/datums/martial/holy_crunch.dm]: D

File: code/datums/martial/holy_crunch.dm [hunk 16]
```diff
@@ -0,0 +1,47 @@
...
+		add_logs(A, D, " used [atk_verb] (Hand of God) on")
+		return 1
```
Comment (kevinz000) [inline code/datums/martial/holy_crunch.dm]: TRUE

File: code/game/objects/items/holy_weapons.dm [hunk 17]
```diff
@@ -336,6 +336,41 @@
...
+		return
+	var/resolve = "<span class='danger'>A phantom force punches your gut!</b></span>"
+	to_chat(user, resolve)
+	if(is_servant_of_ratvar(user) || iscultist(user))
+		to_chat(user, "<span class='redtext'><b>\"YOU HAVE BEEN DEEMED UNWORTHY, CULTIST SCUM!\"</b></span>")
+		user.dropItemToGround(/obj/item/nullrod/hand_of_god)
+		user.status_flags |= DISFIGURED
+		user.bleed_rate = 5
+		user.gib_animation()
+		sleep(3)
```
Comment (kevinz000) [inline code/game/objects/items/holy_weapons.dm]: no sleeps

**Comment by QualityVan** [comment]
why did you rename this
Holy Crunch! was way better

**Comment by Bawhoppen** [comment]
I agree

File: code/datums/martial/holy_crunch.dm [hunk 18]
```diff
@@ -0,0 +1,47 @@
...
+	if(attack_type == "toxin")
+		var/atk_verb = pick("power word: impair", "power word: afflict", "power word: debilitate", "power word: hurt", "power word: diminish", )
+		D.visible_message("<span class='danger'>[A] uses [atk_verb] on [D]!</span>", \
+					  "<span class='userdanger'>[A] uses [atk_verb] on you!</span>")
+		D.apply_damage(10, TOX)
+		playsound(get_turf(D), 'sound/effects/bamf.ogg', 25, 1, -1)
+		add_logs(A, D, " used [atk_verb] (Hand of God) on")
+		return 1
+	if(attack_type == "knockdown")
+		var/atk_verb = pick("power word: stasis", "power word: daze", "power word: tranquilise", "power word: benumb", "power word: ban", )
```
Comment (DoctoreMad) [inline code/datums/martial/holy_crunch.dm]: Power word: Ban? That's some OOC in IC if I've seen it before.

Comment (ma44) [reply code/datums/martial/holy_crunch.dm]: What, are you saying spessmen never played a game involving the galaxy wide internet to not know the word 'ban'?

**Comment by Armhulen** [comment]
Because holy crunch makes no sense and hand of god is a pretty funny reference

File: code/datums/martial/holy_crunch.dm (hunk 18 above)
Comment (Armhulen) [reply code/datums/martial/holy_crunch.dm]: ban
officially or legally prohibit.

or even better

archaic
a curse.

**Comment by QualityVan** [comment]
also isn't there already a chaplain rod skin called God Hand

**Comment by jughu** [comment]
This is not god hand
This is hand OF god

**Comment by ExcessiveUseOfCobblestone** [comment]
if you call it hand of god then there becomes a problem with actually adding hand of god back.

you shall not pass!

File: code/datums/martial/holy_crunch.dm (hunk 13 above)
Comment (Armhulen) [reply code/datums/martial/holy_crunch.dm]: ??

**Comment by Armhulen** [comment]
okay fine holy crunch it is 👀

**Comment by Armhulen** [comment]
i'll change that later

**Comment by KorPhaeron** [comment]
cereal lmao

**Comment by Armhulen** [comment]
he will not divide us

File: code/datums/martial/holy_crunch.dm (hunk 18 above)
Comment (DoctoreMad) [reply code/datums/martial/holy_crunch.dm]: Those arguments are really lukewarm, but I guess I won't be turning any heads anyways.

**Comment by MoreRobustThanYou** [comment]
i want holy crunch

**Comment by factoryman942** [comment]
make it give you a cereal box that you have to take to chef and eat from a bowl with milk

**Comment by Armhulen** [comment]
You could learn the martial art from the holy crunch box

**Comment by Armhulen** [comment]
But I'd need sprites

**Comment by QualityVan** [comment]
make sure the box also has a free toy

**Comment by ExcessiveUseOfCobblestone** [comment]
you could call it Chapoeira as a reference to [Capoeira](https://en.wikipedia.org/wiki/Capoeira)

File: code/datums/martial/holy_crunch.dm (hunk 13 above)
Comment (Cyberboss) [reply code/datums/martial/holy_crunch.dm]: playsound(D, ...

File: code/game/objects/items/holy_weapons.dm (hunk 17 above)
Comment (Cyberboss) [reply code/game/objects/items/holy_weapons.dm]: use add timer

File: code/game/objects/items/holy_weapons.dm [hunk 19]
```diff
@@ -336,6 +336,41 @@
...
+		user.spread_bodyparts()
+		return 1
+	user.Knockdown(80)
+	var/message = "<span class='sciradio'>You have been deemed worthy by the powers above, and now may control a fraction of it's power! Your attacks will switch between brute, burn, toxin, and a knockdown! <b>Deus Vult!</b></span>"
+	to_chat(user, message)
+	var/datum/martial_art/hand_of_god/handofgod = new(null)
+	handofgod.teach(user)
+	user.drop_item()
+	visible_message("<span class='warning'>[src] lights up in fire and quickly burns to ash.</span>")
+	new /obj/effect/decal/cleanable/ash(get_turf(src))
```
Comment (Cyberboss) [inline code/game/objects/items/holy_weapons.dm]: drop_location()

File: code/game/objects/items/holy_weapons.dm [hunk 20]
```diff
@@ -336,6 +336,41 @@
...
+		user.spawn_gibs()
+		user.spill_organs()
+		user.spread_bodyparts()
+		return 1
+	user.Knockdown(80)
+	var/message = "<span class='sciradio'>You have been deemed worthy by the powers above, and now may control a fraction of it's power! Your attacks will switch between brute, burn, toxin, and a knockdown! <b>Deus Vult!</b></span>"
+	to_chat(user, message)
+	var/datum/martial_art/hand_of_god/handofgod = new(null)
+	handofgod.teach(user)
+	user.drop_item()
```
Comment (Cyberboss) [inline code/game/objects/items/holy_weapons.dm]: if you expect the user to be dropping src why qdel it later?

File: code/game/objects/items/holy_weapons.dm [hunk 21]
```diff
@@ -336,6 +336,41 @@
...
+		user.dropItemToGround(/obj/item/nullrod/hand_of_god)
+		user.status_flags |= DISFIGURED
+		user.bleed_rate = 5
+		user.gib_animation()
+		sleep(3)
+		user.adjustBruteLoss(1000)
+		user.spawn_gibs()
+		user.spill_organs()
+		user.spread_bodyparts()
+		return 1
```
Comment (Cyberboss) [inline code/game/objects/items/holy_weapons.dm]: attack_self has no return value iirc

File: code/game/objects/items/holy_weapons.dm (hunk 19 above)
Comment (Armhulen) [reply code/game/objects/items/holy_weapons.dm]: very nice

**Comment by Armhulen** [comment]
@ExcessiveUseOfCobblestone i actually like this more than holy crunch or hand of god happening

**Comment by Armhulen** [comment]
then again i can't think of a martial art that uses magic in real life :thought_balloon:

**Comment by Armhulen** [comment]
FUCK I'M REWORKING IT AGAIN >:(

**Comment by Repukan** [comment]
Add Hamon

**Comment by Doppeltongue** [comment]
Maybe call it THE TOUCH OF GOD! and make it lightning themed?

**Comment by Armhulen** [comment]
HOLY

**Comment by Armhulen** [comment]
CRUNCH

File: code/datums/martial/holy_crunch.dm (hunk 13 above)
Comment (Armhulen) [reply code/datums/martial/holy_crunch.dm]: oh i see

File: code/datums/martial/holy_crunch.dm (hunk 14 above)
Comment (Armhulen) [reply code/datums/martial/holy_crunch.dm]: it's 1/4 to test your brute damage (1/4 to add 10% knockdown chance) and if you don't get the knockdown it's a wasted attack

Comment (Armhulen) [reply code/datums/martial/holy_crunch.dm]: and your 10 free attacks will be plagued by useless knockdown attacks

File: code/datums/martial/holy_crunch.dm (hunk 5 above)
Comment (Armhulen) [reply code/datums/martial/holy_crunch.dm]: works for everything but stamina hmm

**Comment by Armhulen** [comment]
i'm gonna fix this some other time

**Comment by DoctoreMad** [comment]
If you weren't such a cornflake this would've gone fine

**Pull request closed without merging**